Fushigi/ui/widgets/CourseScene.cs

```diff
@@ -2335,6 +2405,42 @@ private async Task<(CourseActor?, KeyboardModifier modifiers)> PickLinkDestInVie
             batchAction.Commit($"{IconUtil.ICON_TRASH} {actionName}");
         }
 
+        //TODO making this undoable
```

Comment:
How about making it **do** something in the first place?
I assume that's what you meant but I feel like the TODO comment should reflect that.

Also we should **not** expose functionality to users that isn't implemented yet.
At the VERY least put it in the title of the Warning Dialog => **Deletion warning (NOT IMPLEMENTED)**
or something like that

Reply:
I wanted to make deleting layers undoable, and now that you mention it, I do not know where that part went. I didn't want to implement the undo action half finished so I deleted the action commit, but I think I deleted the layer removal by mistake too. Regardless I figured out how to get the undo to work, I learned yesterday that assigning dictionaries to another variables did not work how I thought.